libs/partners/zenguard/pyproject.toml

```diff
@@ -0,0 +1,85 @@
+[tool.poetry]
+name = "langchain-zenguard"
+version = "0.1.0"
+description = "An integration package connecting Zenguard and LangChain"
+authors = []
+readme = "README.md"
+repository = "https://github.com/langchain-ai/langchain"
```

Comment:
wait, this should point to ZenGuard repo?

Reply:
in the end our integration will be in the langchain repository so that's correct